### ISSUE TITLE
v8.x.x Breaks cssnext somehow

### ISSUE THREAD

**Issue by JonathanWolfe** [postcss/postcss-import#162]
After updating to `8.0.1` this morning `color()` functions via `postcss-cssnext` began failing to compile if they included a `var()` as the color reference. Rolling back to `postcss-import@<8.0.0` fixed the compiling issue.

__postcss.config.json__
```json
{
	"input": "./src/css/main.css",
	"output": "./compiled/css/bundle.min.css",
	"use": [ "postcss-import", "postcss-cssnext", "css-mqpacker", "cssnano" ],
	"local-plugins": true,
	"cssnano": {
		"discardComments": {
			"removeAll": true
		}
	}
}
```
__package.json__
```json
"devDependencies": {
	"css-mqpacker": "latest",
	"cssnano": "latest",
	"postcss-cli": "latest",
	"postcss-cssnext": "latest",
	"postcss-import": "<8.0.0"
},
```

__Error when using `v8.0.1`__
```
postcss --config postcss.config.json
{ [Error: /Users/jonwolfe/repos/hopspot/src/css/base/_buttons.css:53:3: Unable to parse color from string "var(--purple)"]
  originalMessage: 'Unable to parse color from string "var(--purple)"',
  fileName: '/Users/jonwolfe/repos/hopspot/src/css/base/_buttons.css',
  lineNumber: 53,
  columnNumber: 3 }
child_process.js:507
    throw err;
    ^
```
__buttons.css (lines 52-54)__
```css
&:active {
	background-color: color(var(--purple) blackness(+10%)) !important;
}
```

**Comment by TrySound** [comment]
Do you have stacktrace?

**Comment by JonathanWolfe** [comment]
Nope. I'll get one.

**Comment by TrySound** [comment]
Show how you import partials

**Comment by JonathanWolfe** [comment]
Here's the css folder itself: [css.zip](https://github.com/postcss/postcss-import/files/106901/css.zip)

**Comment by TrySound** [comment]
Same problem with comments.
https://github.com/postcss/postcss-import/issues/163

**Comment by MoOx** [comment]
Closed by #164

**Comment by MoOx** [comment]
8.0.2 released
